app/helpers/atmosphere/virtual_machine_flavors_helper.rb

```diff
@@ -0,0 +1,11 @@
+module Atmosphere
+  module VirtualMachineFlavorsHelper
+    def each_os_family(&block)
```

Comment:
Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used. You can also write as `each_os_family(*)` if you want the method to accept any arguments but don't care about them.